### DISCUSSION TITLE
How to include php into tpl file??

### DISCUSSION THREAD

**Discussion by Jianbin719** [YetiForceCompany/YetiForceCRM#17226] [How to]
I would like to add functions of "ProcessWizard.php" into the progress bar(layouts/basic/modules/Vtiger/Detail/HeaderProgress.tpl)

The features I want. to add clickable button to the progress bar. also required field will come as pop up windows when I change the status of record. 

So how to include php into this tpl file??

**Comment by mariuszkrzaczkowski** [comment]
https://github.com/YetiForceCompany/YetiForceCRM/discussions/15114

**Comment by Jianbin719** [comment]
sorry, I know the code in https://github.com/YetiForceCompany/YetiForceCRM/discussions/15114

I mean to include ProcessWizard code into the progress bar. like it below. 

![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/75652286/80394122-324c-4e21-96ea-130c6bce61ed)

**Comment by Jianbin719** [comment]
I have this question, because there is bug in the progress bar, I think so. 

I want something like, when the ticket is closed, the resolution/result field is required (set it Dependent Fields). But in real operation, the required filed could be ignored when change the state from progress bar directly.
